pkg/sentry/fsimpl/fuse/connection.go

```diff
@@ -63,6 +76,159 @@ type futureResponse struct {
 // Connection is the struct by which the sentry communicates with the FUSE server daemon.
 type Connection struct {
 	fd *DeviceFD
+
+	// Initialized after receiving FUSE_INIT reply.
+	// Until it's set, suspend sending FUSE requests.
+	Initialized bool
+
+	// protects Initialized.
+	initializedLock sync.Mutex
```

Comment:
I think you can use atomic.LoadInt32 and atomic.StoreInt32 and you will not need initializedLock in this case.